src/tools/work-items.ts

```diff
@@ -71,14 +71,22 @@ function configureWorkItemTools(server: McpServer, tokenProvider: () => Promise<
       team: z.string().describe("The name or ID of the Azure DevOps team."),
     },
     async ({ project, team }) => {
-      const connection = await connectionProvider();
-      const workApi = await connection.getWorkApi();
-      const teamContext = { project, team };
-      const backlogs = await workApi.getBacklogs(teamContext);
-
-      return {
-        content: [{ type: "text", text: JSON.stringify(backlogs, null, 2) }],
-      };
+      try {
```

Comment:
consider introducing a utility method that does the `try-catch`  and takes a `handler` and `error message`, this could be re-used for all of these tools instead of explicitly adding a try-catch in each tool